.gitignore

```diff
@@ -5,4 +5,5 @@ node-api-postgres/pool.js
 node-api-postgres/node_modules
 .next
 id_rsa
+id_rsa.pub
```

Comment:
this change should not be here... not sure how or why it got added

Reply:
It was intentional as there was `id_rsa.pub` sitting on my local repo form when we set up the backend on my machine.

Note: This is reverted. As explained by @sdevalapurkar : there is no `id_rsa.pub` file in the repo so it doesn't need to be in `.gitignore`